portals/getting_started.rst

```diff
@@ -5,14 +5,10 @@
 Getting Started
 ###############
 
-Welcome to Citus. As you begin it helps to take some time to get oriented.
+Welcome to Citus!
 
-* For a short description, along with a brief discussion of use-cases see, :ref:`what_is_citus` This is a good place to determine how Citus fits your needs.
-* Then read :ref:`introduction_to_citus` to get an overview of how the parts fit together. The terms described there are used throughout the documentation, so it's best to become familiar with them.
-
-If you've found that Citus looks good for your use-case, the best way to learn more is to try it for yourself. The extension is available both as an open-source PostgreSQL extension that you can install on your own machine, or as managed Database-as-a-Service.
-
-* A great place to start experimenting with Citus is a local :ref:`development` environment. That section explains how to install Citus on various platforms, or use Docker depending on your preference.
-* Once you have Citus installed, try our ten-minute tutorials. One for :ref:`multi_tenant_tutorial` and another for :ref:`real_time_analytics_tutorial`.
-
-After giving Citus a try, you can look at more full featured example applications for Citus :ref:`use_cases_topic`.
+* :ref:`what_is_citus` - Citus use-cases. Do they match yours?
+* :ref:`introduction_to_citus` - How the parts fit together.
+* :ref:`single_machine_docker` - Install instructions to try it locally.
+* :ref:`multi_tenant_tutorial` / :ref:`real_time_analytics_tutorial` - Short tutorials.
+* :ref:`use_cases_topic` - Bigger example of use.
```

Comment:
Instead of "Bigger example", maybe something else? Like "In-depth use-case guides and examples"?